lib/oktakit/client.rb

```diff
@@ -48,18 +48,26 @@ def initialize(token:, organization:)
     # @return [Sawyer::Resource]
     def get(url, options = {})
       should_paginate = options.delete(:paginate)
-      resp, status, next_page = request :get, url, query: options.delete(:query),
-                                                   headers: options.delete(:headers),
-                                                   accept: options.delete(:accept),
-                                                   content_type: options.delete(:content_type),
-                                                   paginate: should_paginate, data: options
+      request_options = {
+        query: options.delete(:query),
+        headers: options.delete(:headers),
+        accept: options.delete(:accept),
+        content_type: options.delete(:content_type),
+        paginate: should_paginate,
+        data: options
+      }
+
+      resp, status, next_page = request :get, url, **request_options
+      return [resp, status] unless status == 200
```

Comment:
I don't quite get this part here? Can you add a comment explaining why we'd return early?